DataFormats/HGCalReco/interface/Trackster.h

```diff
@@ -36,13 +37,28 @@ namespace ticl {
     // created the trackster. For track-based seeding the pointer to the track
     // can be cooked using the previous ProductID and this index.
     int seedIndex;
+    // We also need the pointer to the original seeding region ??
```

Comment:
is this a todo for later, or should it be addressed in this PR?

Reply:
It's for later, but better leave it there.